src/views/preview/comment/compose-comment.jsx

```diff
@@ -118,7 +118,7 @@ class ComposeComment extends React.Component {
     }
 
     isMuted () {
-        return this.state.muteExpiresAt * 1000 > Date.now();
```

Comment:
muteExpiresAt in the state object is already being saved in ms in the constructor and in the comment's post response so I'm pretty sure this one is a bug.